components/de_amp/common/pref_names.cc

```diff
@@ -9,7 +9,7 @@
 
 namespace de_amp {
 
-const char kDeAmpPrefEnabled[] = "brave.de_amp.enabled";
```

Comment:
Why do we need this change?
It results to "reseting" the pref for uses who have set the old pref.